prow/plugins/approve/approve_test.go

```diff
@@ -155,6 +156,8 @@ func (fr fakeRepo) IsNoParentOwners(path string) bool {
 func TestHandle(t *testing.T) {
 	// This function does not need to test IsApproved, that is tested in approvers/approvers_test.go.
 
+	var githubLinkURL string = "https://github.com"
```

Comment:
Why not provide the literal string in the test cases? No need to connect cases together right?

Reply:
Done, switched to static string. :smile: